lldb/packages/Python/lldbsuite/test/tools/lldb-dap/lldbdap_testcase.py

```diff
@@ -333,6 +333,7 @@ def attach(
         exitCommands=None,
         attachCommands=None,
         coreFile=None,
+        stopOnAttach=True,
```

Comment:
Should we default this to `False`?

Or should we have the `DAPTestCaseBase` case `attach` and `launch` helpers take an `initialSourceBreakpoints, initialFunctionBreakpoints, initialExceptionBreakpoints` parameter and we help the tests by sending these as part of the setup flow, kind of how we have `disconnectAutomatically` as a flag to handle shutdown flows automatically.

Reply:
The way the tests are currently written, they expect to be stopped after the launch (presumably because these tests were never sending the `configurationDone` request). I went with this default to keep the changes minimal, but I like the idea of the initial breakpoints as a way to improve this in a subsequent PR.

Reply:
Sounds good, we can follow up with some other improvements to tests.